contract/call/src/calls.rs

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019 Parity Technologies (UK) Ltd.
+// This file is part of ink!.
+//
+// ink! is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// ink! is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with ink!.  If not, see <http://www.gnu.org/licenses/>.
+
+use crate::{AccountId, AccountIndex, Balance, Hash, NodeRuntimeTypes};
+use ink_core::env::EnvTypes;
+// use ink_core::storage::Vec;
+use ink_prelude::vec::Vec;
+use pallet_indices::address::Address;
+use scale::{Codec, Decode, Encode};
+use sp_runtime::traits::Member;
+
+/// Default runtime Call type, a subset of the runtime Call module variants
+///
+/// The codec indices of the  modules *MUST* match those in the concrete runtime.
+#[derive(Encode, Decode)]
+#[cfg_attr(feature = "std", derive(Clone, PartialEq, Eq))]
+pub enum Call {
+    #[codec(index = "6")]
+    Balances(Balances<NodeRuntimeTypes, AccountIndex>),
```

Comment:
We don't the code. I didn't remove it because the whole end to end flow not verified yet, I mean in runtime node. The most basic code really help to check our implementation and debug.